src/Microsoft.ML.AutoML/AutoMLExperiment/IDatasetSettings.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.ML.AutoML
+{
+    internal interface IDatasetSettings
+    {
+    }
+
+    internal class TrainTestDatasetSettings : IDatasetSettings
+    {
+        public IDataView TrainDataset { get; set; }
+
+        public IDataView TestDataset { get; set; }
+    }
```

Comment:
Is there a reason to use this over the `TrainTestSplit` we already have?

Reply:
Nope, it's just possible that train set and test set are not from the same datasource

Reply:
Thats a good point. I hadn't thought of that.